XP/install/assets.json

```diff
@@ -22,8 +22,8 @@
         {
             "id": "spe",
             "name": "Sitecore PowerShell Extensions",
-            "fileName": "Sitecore PowerShell Extensions-5.0 for 9.2.scwdp.zip",
-            "url": "https://dev.sitecore.net/~/media/7AB15C0432624E6CA9786DF00DD653AB.ashx",
+            "fileName": "Sitecore.PowerShell.Extensions-5.1.scwdp.zip",
+            "url": "https://github.com/SitecorePowerShell/Console/releases/download/5.1/Sitecore.PowerShell.Extensions-5.1.scwdp.zip",
```

Comment:
According to https://dev.sitecore.net/Downloads/Sitecore_Experience_Accelerator/19/Sitecore_Experience_Accelerator_190.aspx
The SPE version should be 5.0 instead of 5.1.